app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -446,7 +446,7 @@ def center_toolbar_filename_classic
               ems_container ems_middleware container_project container_route container_replicator container_image
               ems_network security_group floating_ip cloud_subnet network_router
               container_image_registry ems_infra flavor host container_build
-              ontap_file_share ontap_logical_disk container_topology middleware_topology
+              ontap_file_share ontap_logical_disk container_topology middleware_topology middleware_server
```

Comment:
I think that once this approach is reviewed I need to rebase or merge. 
As we are working with several different branches on the same provider is possible that we modify same files and we might create some conflicts.
Anyway I counted with some mergeable tasks of all work once this is done.
I wonder if for the hawkular provider it makes sense to create a branch that we could consolidate in two steps, one putting together all hawkular related efforts, and a second trying to consolidate in MiQ.
I don't know I guess that a hawkular branch can save us for conflicts within the provider, but perhaps is adding a little bit more overhead.